metaserver/src/main/scala/scala/meta/languageserver/providers/DocumentFormattingProvider.scala

```diff
@@ -79,14 +78,11 @@ class DocumentFormattingProvider(
       }
     }
     formatResult match {
```

Comment:
😇 

```scala
import cats.syntax.bifunctor._
import cats.instances.either._

formatResult.bimap(
  Response.invalidParams,
  formatted => List(TextEdit(fullDocumentRange, formatted))
)
```

Reply:
TIL! Done.

Reply:
It's a shame the stdlib `Either` does not have a signature like that, it's so close to `fold` but different enough